### ISSUE TITLE
Docs: explaining of HTTP message flow format

### ISSUE THREAD

**Issue by vetss** [RestComm/gmlc#21]
We need to explain the format of HTTP part message format. HTTP POST message flow (MLP format) is not yet proper implemented at the server side (it always return fake values), so let's do not add this part into a a manual this step.
GET requests work now and we need do describe at least this part.

**Comment by vetss** [comment]
GET request template:

`http://<IP>:8080/restcomm?msisdn=59899077937`

GET response examples:

`mcc=250,mnc=1,lac=32000,cellid=221,aol=5,vlrNumber=5555555666`
(success response)
`SubscriberState: SubscriberState [subscriberStateChoice=netDetNotReachable, notReachableReason=notRegistered]`
(returnResultLast with only SubscriberState without LocationInformation)
`Unknown SubscriberInfo received: xxxx`
(bad returnResultLast)
`Unknown AnyTimeInterrogationResponse received: xxxx`
(bad returnResultLast)
`ReturnError: 1 : MAPErrorMessageUnknownSubscriber [, unknownSubscriberDiagnostic=imsiUnknown]`
(UnknownSubscriber error response)
`ReturnError: <error code> : <MAP Error message description>`
`ReturnError: 34 : MAPErrorMessageSystemFailure [networkResource=hlr]`
(received othe MAP Error message)
`Invalid MSISDN specified`
(in the case when MSISDN is absent in the GET HTTP request - bad HTTP request syntax)
`DialogTimeout`
(usually because of no response from an HLR)
`DialogReject: <description>`
`DialogProviderAbort: <description>`
`DialogUserAbort: <description>`
`RejectComponent: <description>`
(these are other MAP protocol errors)
`System Failure: <description>`
(some message processing failure occured at the GMLC server)

**Comment by vetss** [comment]
POST request template:

```
<?xml version="1.0" encoding="UTF-8"?>
 <!DOCTYPE svc_init SYSTEM "MLP_SVC_INIT_310.dtd">
 <svc_init xmlns="MLP_SVC_INIT_310.dtd">
   <hdr>
     <client>
       <id>USERNAME</id>
       <pwd>PASSWORD</pwd>
       <serviceid>SERVICEID</serviceid>
     </client>
   </hdr>
   <slir>
     <msids>
       <msid type="MSISDN">12345</msid>
     </msids>
     <eqop>
        <resp_timer>15</resp_timer>
     </eqop>
   </slir>
</svc_init>
```

succeeded response:

```
<?xml version="1.0" encoding="UTF-8"?><!DOCTYPE svc_result SYSTEM "MLP_SVC_RESULT_310.DTD">
<svc_result xmlns="MLP_SVC_RESULT_310.dtd" ver="3.1.0">
    <slia ver="3.1.0">
        <pos>
            <msid>12345</msid>
            <pd>
                <time utc_off="+0100">20160324123959</time>
                <shape>
                    <CircularArea>
                        <coord>
                            <X>27 28 25.00S</X>
                            <Y>153 01 43.00E</Y>
                        </coord>
                        <radius>5000</radius>
                    </CircularArea>
                </shape>
            </pd>
        </pos>
    </slia>
</svc_result>
```

error responses - UNKNOWN SUBSCRIBER

```
<?xml version="1.0" encoding="UTF-8"?><!DOCTYPE svc_result SYSTEM "MLP_SVC_RESULT_310.DTD">
<svc_result xmlns="MLP_SVC_RESULT_310.dtd" ver="3.1.0">
    <slia ver="3.1.0">
        <result resid="4">UNKNOWN SUBSCRIBER</result>
        <add_info>ReturnError: 1 : MAPErrorMessageUnknownSubscriber [, unknownSubscriberDiagnostic=imsiUnknown]</add_info>
    </slia>
</svc_result>
```

error responses - SYSTEM FAILURE (in other cases)

```
<?xml version="1.0" encoding="UTF-8"?><!DOCTYPE svc_result SYSTEM "MLP_SVC_RESULT_310.DTD">
<svc_result xmlns="MLP_SVC_RESULT_310.dtd" ver="3.1.0">
    <slia ver="3.1.0">
        <result resid="1">SYSTEM FAILURE</result>
        <add_info>ReturnError: 34 : MAPErrorMessageSystemFailure [networkResource=hlr]</add_info>
    </slia>
</svc_result>
```

**Comment by deruelle** [comment]
Part of Issue #52

**Comment by FerUy** [comment]
Work started in issue #52 as for commit 1032166

**Comment by FerUy** [comment]
Closed as for commit 5e52dc8.
